mycoffee/params.py

```diff
@@ -432,3 +432,5 @@
 FILE_FORMATS_LIST = ["text", "json"]
 
 MODES_LIST = ["water-to-coffee", "coffee-to-water"]
+
+MODES_NAME_MAP = {"water-to-coffee": "Water --> Coffee", "coffee-to-water": "Coffee --> Water"}
```

Comment:
It should be `MODE_NAMES_MAP` I think

Reply:
or `MODE_TO_NAME`

Reply:
fixed in https://github.com/sepandhaghighi/mycoffee/pull/132/commits/f17baf12a6647802338b0980e397c4f7e5e17b7f